po/de.po

```diff
@@ -2799,7 +2792,7 @@ msgstr ""
 "Modul aus dem Motif-Code in Version 1.1 herausgelöst. Wie man in der "
 "Statistik sieht, hat sich das Register als ziemlich stabil erwiesen. Ab "
 "Version 1.7 zeigt die Zelle einen 2. Eintrag: die Anzahl der Codezeilen in "
-"src/register/register-gnome, welches zuvor als Teil von Gnome gezählt wurde."
+"src/register/register-gnome, welches zuvor als Teil von Gnome gezählt wurde"
```

Comment:
Findest du nicht, daß da in https://www.gnucash.org/sizing.phtml ein Satzzeichen hingehört?